tests/py/dynamo/conversion/harness.py

```diff
@@ -214,75 +193,27 @@ def generate_graph(
         self,
         mod: torch.nn.Module,
         original_inputs: List[torch.Tensor],
-        expected_ops: Set[Callable],
-        unexpected_ops: Optional[Set[Callable]] = None,
-        customized_passes: List[Callable] = None,
-        disable_passes: bool = False,
     ):
-        # Torchdynamo+aot proxytensor tracer
-        # Below are common passes
-        passes_list = [
-            compose_bmm,
-            compose_chunk,
-            compose_getitem_slice,
-            replace_aten_reshape_alias_with_replace,
-            replace_aten_op_with_indices,
-            replace_transpose_mm_op_with_linear,  # after compose_bmm
-            replace_native_layernorm_with_layernorm,
-            remove_ops,
-            replace_builtin_ops,  # after replace_native_layernorm_with_layernorm
-        ]
-        # Combine with customized passes specific to any model
-        if customized_passes:
-            passes_list.extend(customized_passes)
-
-        if disable_passes:
-            passes_list = []
-
-        fx_module, _ = aten_tracer.trace(mod, original_inputs)
-        for passes in passes_list:
-            pr: PassResult = passes(fx_module)
-            fx_module = pr.graph_module
-        fx_module(*original_inputs)
-
-        fx_module = run_const_fold(fx_module)
+        fx_module = torch.fx.symbolic_trace(mod)
```

Comment:
Consider switching to:
```python
fx_module = torch._dynamo.export(mod, *original_inputs,aten_graph=True, assume_static_by_default=True, tracing_mode="real").graph_module
```
I think we just need light `ATen` tracing, which is where this function can help resolve many issues.